mobile/lib/modules/home/views/home_page.dart

```diff
@@ -13,6 +14,10 @@ import 'package:immich_mobile/modules/home/ui/profile_drawer.dart';
 import 'package:immich_mobile/shared/providers/asset.provider.dart';
 import 'package:immich_mobile/shared/providers/server_info.provider.dart';
 import 'package:immich_mobile/shared/providers/websocket.provider.dart';
+import 'package:openapi/api.dart';
+
+import '../../../constants/hive_box.dart';
```

Comment:
I would prefer to import the file like the format above `package:immich_mobile...`